src/App.test.js

```diff
@@ -1,8 +1,39 @@
-import { render, screen } from '@testing-library/react';
-import App from './components/App';
+import operate from './logic/operate';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe('operate', () => {
+  test('adds two numbers', () => {
+    expect(operate('3', '5', '+')).toBe('8');
+  });
+
+  test('subtract two numbers', () => {
+    expect(operate('4', '2', '-')).toBe('2');
+  });
+
+  test('multiply two numbers', () => {
+    expect(operate('5', '5', '×')).toBe('25');
+  });
+
+  test('divdide two numbers', () => {
+    expect(operate('6', '2', '÷')).toBe('3');
+  });
+
+  test('divdide by zero', () => {
+    expect(operate('6', '0', '÷')).toBe('Can\'t divide by 0.');
+  });
+
+  test('modulus operation', () => {
+    expect(operate('7', '2', '%')).toBe('1');
+  });
+
+  test('modulus operation by zero', () => {
+    expect(operate('7', '0', '%')).toBe('Can\'t find modulo as can\'t divide by 0.');
+  });
+
+  test('multiply by zero', () => {
+    expect(operate('6', '0', '×')).toBe('0');
+  });
+
+  test('modulus operation', () => {
+    expect(operate('9', '6', '%')).toBe('3');
+  });
 });
```

Comment:
[REQUIRED] I don't think there is any point in testing the modulus operator twice.